packages/cli/src/services/abstract-check-runner.ts

```diff
@@ -173,7 +175,13 @@ export default abstract class AbstractCheckRunner extends EventEmitter {
     Array.from(this.checks.entries()).forEach(([checkRunId, { check }]) =>
       this.timeouts.set(checkRunId, setTimeout(() => {
         this.timeouts.delete(checkRunId)
-        this.emit(Events.CHECK_FAILED, check, `Reached timeout of ${this.timeout} seconds waiting for check result.`)
+        let errorMessage = `Reached timeout of ${this.timeout} seconds waiting for check result.`
+        // Checkly should always report a result within 240s.
+        // If the default timeout was used, we should point the user to the status page and support email.
+        if (this.timeout === DEFAULT_CHECK_RUN_TIMEOUT_SECONDS) {
```

Comment:
Related with previous comment: if the user set `--timeout=300`, in case the check doesn't `FINISH`, he will get the `Checkly may be experiencing problems.` message.

Reply:
If a check takes longer than 240s, the Checkly backend should normally be timing out/cancelling the check run and reporting a result to the CLI. This timeout in the CLI is mainly a fallback. If the CLI doesn't receive a result in 300 seconds, then it's likely because something went wrong on the Checkly side.